.travis.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       script: ./gradlew check --info
     - stage: sonar
       script: ./gradlew jacocoTestReport sonarqube --info
+    - stage: integration
+      script: ./gradlew integrationTest
```

Comment:
Run the integration tests last, so that sonar results come out quicker, as the integration tests take 10 mins or so to complete.

Run the integration tests without `--info`, as the logging is very verbose, and sometimes exceeds the amount of output that Travis allows, resulting in the job being terminated.